### PR TITLE
fix(Map): use same size calculation in `updateSize` and `updateViewportSize_`

### PR DESCRIPTION
**Pull request by simonseyock**
Fixes: #15268

### CODE REVIEW

File: src/ol/View.js [hunk 1]
```diff
@@ -74,9 +74,11 @@ import {fromExtent as polygonFromExtent} from './geom/Polygon.js';
...
 /**
  * @typedef {Object} FitOptions
- * @property {import("./size.js").Size} [size] The size in pixels of the box to fit
- * the extent into. Default is the current size of the first map in the DOM that
- * uses this view, or `[100, 100]` if no such map is found.
+ * @property {import("./size.js").Size} [size] The size in pixels of the box to
+ * fit the extent into. If the view is associated with one map, it defaults to
+ * the size of the map. If it is associated with multiple maps, the size can be
+ * of any one of the maps, so the desired size should be specified. If no map
+ * is associated it uses `[100, 100]` as the size.
```
Comment (ahocevar) [inline src/ol/View.js:81]:
```suggestion
 * @property {import("./size.js").Size} [size] The size in pixels of the box to
 * fit the extent into. Defaults to the size of the map the view is associated with.
 * If no map or multiple maps are connected to the view, the desired box size
 * (e.g. `map.getSize()`) should be provided.
```

**Review by ahocevar — CHANGES_REQUESTED** [review]
Good simplification. Thanks, @simonseyock. I have added a suggestion for simpler wording that you may want to consider. Looks good otherwise.

File: src/ol/View.js (hunk 1 above)
Comment (simonseyock) [reply src/ol/View.js]: I changed `should be provided` to `has to be provided to get a usable result`.

**Comment by simonseyock** [comment]
Just to clarify: This not only simplifies it, but also fixes a bug that was occurring in the removed code part.

**Comment by simonseyock** [comment]
Also I updated the docs for `calculateExtent` with the same text.

**Review by ahocevar — APPROVED** [review]
Thanks, @simonseyock

**Pull request merged**